devices/xiaomi.js

```diff
@@ -2403,6 +2527,29 @@ module.exports = [
             await endpoint.read('aqaraOpple', [0x040a], {manufacturerCode: 0x115f});
         },
     },
+    {
+        zigbeeModel: ['aqara.feeder.acn001'],
+        model: 'ZNCWWSQ01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara Pet Feeder C1',
+        fromZigbee: [fzLocal.aqara_feeder],
+        toZigbee: [tzLocal.aqara_feeder],
+        exposes: [
+            exposes.enum('feeding', ea.STATE_SET, ['START']).withDescription('Start feeding'),
+            exposes.text('feeding_source', ea.STATE).withDescription('Feeding source'),
```

Comment:
This should be an enum (`manual`/`remote`)

Reply:
yes... wait